### PR TITLE
User/raych1/new mv

### PR DESCRIPTION
**Pull request by raych1**
adding new version of model validation

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -30,8 +30,8 @@
     // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
 
     /* Additional Checks */
-    "noUnusedLocals": true /* Report errors on unused locals. */,
-    "noUnusedParameters": true /* Report errors on unused parameters. */,
+    "noUnusedLocals": false /* Report errors on unused locals. */,
```
Comment (ruowan) [inline tsconfig.json]: Why change this?

Comment (raych1) [reply tsconfig.json]: reverted this change.

**Comment by ruowan** [comment]
Why a new version of model validation? Can you explain the functionalities added in this PR?

**Comment by raych1** [comment]
> Why a new version of model validation? Can you explain the functionalities added in this PR?

It removes the dependency on sway and z-schema then uses AJV to do JSON validation. This aligns with live validation.

**Review by ruowan — APPROVED** [review]

**Pull request merged**